ts-client/package.json

```
@@ -0,0 +1,30 @@
+{
+  "name": "@striveworks/valor",
+  "description": "A TypeScript API client for Valor.",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "build": "tsc",
+    "test": "jest"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/striveworks/valor.git"
+  },
+  "keywords": [
+    "valor",
+    "striveworks"
+  ],
+  "author": "Striveworks <valor@striveworks.com>",
```

https://docs.npmjs.com/cli/v7/configuring-npm/package-json#people-fields-author-contributors

add ref to striveworks.com? 

```suggestion
  "author": "Striveworks <valor@striveworks.com> (https://www.striveworks.com/)",
```